templates/containers.tmpl

```diff
@@ -11,7 +11,7 @@
         {{- if .Ports}}
         ports:
         {{- range .Ports}}
-        - containerPort: {{.Port}}
+              - containerPort: {{- if .TargetPort }} {{.TargetPort}} {{- else }} {{.Port}} {{- end}}
```

Comment:
Dedent it by 4 spaces please else indent the name on the next line as well.

Reply:
fixed! sorry